validation/mock_trace_validation.go

```diff
@@ -1,24 +1,37 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 package validation
 
 import (
 	"reflect"
 
+	"github.com/golang/protobuf/ptypes"
+
 	"google.golang.org/genproto/googleapis/devtools/cloudtrace/v2"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
-
-	"github.com/golang/protobuf/ptypes"
 )
 
 var (
-	InvalidTimestampErr   = status.Error(codes.InvalidArgument, "start time must be before end time")
-	MalformedTimestampErr = status.Error(codes.InvalidArgument, "unable to parse timestamp")
-	MissingFieldError     = status.New(codes.InvalidArgument, "Missing required field(s)")
-	requiredFields        = []string{"Name", "SpanId", "DisplayName", "StartTime", "EndTime"}
+	InvalidTimestampError   = status.Error(codes.InvalidArgument, "start time must be before end time")
```

Comment:
I think the convention for Go is typically
```ErrInvalidTimestamp = ...
ErrMalformedTimstamp = ...```

See https://golang.org/pkg/io/#pkg-variables

Reply:
Got it, done